src/library_syscall.js

```diff
@@ -224,6 +224,9 @@ var SyscallsLibrary = {
     exit(status);
     return 0;
   },
+  __syscall2: function(which, varargs) { // fork
+    return -{{{ cDefine('ENOSYS') }}};
+  },
```

Comment:
Why is this necessary now but not before? It seems like it would be a better idea to remove or #ifdef out any code that wants to call fork.